src/main/java/Board.java

```diff
@@ -1,4 +1,105 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Board {
-    public static String[] grid = new String[9];
+
+    private List<String> cells = Arrays.asList("", "", "", "", "", "", "", "", "");
+
+    public List<String> getCells() {
+        return cells;
+    }
+
+    public boolean gridIsEmpty() {
+       List<String> currentBoard = getCells();
+       for (int i = 0; i < currentBoard.size(); i++) {
+           if (currentBoard.get(i) != "") {
+               return false;
+           } else {
```

Comment:
Why is *grip* in you method name? I think you check your board content here. 
The name isEmpty (for example) could  enough as you will call board.isEmpty()

If you select the *currentBoard.get(i) != ""* and press OPTION + COMMAND + M you could extract this part of the code, may have a name which express your intention like *isEmptyCell(i)*

You dont need the else condition here.